_posts/2021-12-15-dask-fellow-reflections.md

```diff
@@ -0,0 +1,172 @@
+
+**Being added to the @dask/maintenance team**
+
+There's no point getting notifications if you don't have GitHub permissions to do anything about them. In future I think we should add only people with at least triage or write permissions to the github teams.
+
+**Real time interaction**
+
+- We tried out "Ask a maintainer" office hours for the life science community, but they were poorly attended, so we cancelled this.
+- We added some "Dask social chat" events to the calendar, but they were not very well attended outside of the first few. Most often, zero people attended. (There is another social chat for the Americas/Europe time zones, which is at a more convenient time for most people and might be more popular.)
+
+**Slack**
+
+Slack works well to DM specific people to set up meeting times, etc, but the public channels didn't end up being very useful for me personally.
+
+**Lack of integration with other project teams**
+
+You can only get so much done as a solo developer. We had hoped that I would naturally end up working with teams from several different projects, but this didn't really end up being the case. The `napari` project is an exception to this, and that relationship was well established before starting work for Dask. Perhaps there's something more we could have done here to facilitate more interaction.
+
+## What's next in Dask?
+
+Lots of stuff has happened in Dask, but there is still lots left to do.
+Here is a summary of the next steps for several projects. We'd love it if new people would like to take up the torch and contribute to any of these projects.
+
+Genevieve will be starting a new job next year, you can find her on GitHub [@GeneviveeBuckley](https://github.com/GenevieveBuckley/).
+
+**ITK image compatibility with Dask**
+
+- The next steps for the ITK + Dask project require ITK release candidate 5.3rc3 or above to become available (likely early in 2022).
+- When the release is available, the next step is to try to re-run the code from the original [ITK blogpost](https://blog.dask.org/2019/08/09/image-itk).
+- If there’s still work to be done we’ll need to open issues for the remaining blockers. And if it all works well, we'd like someone to write a second ITK + Dask blogpost to publicize the new functionality.
+
+**Improving performance around rechunking**
+
+More performance improvements related to rechunking is required (see [#7950](https://github.com/dask/dask/pull/7950) and [#7980](https://github.com/dask/dask/pull/7980)).
+
+**High level graph work for arrays and slicing**
```

Comment:
FYI @ian-r-rose
(I might end up quoting some of your notes here, I think you've said things better than me)

Reply:
Sounds good!